src/sap/fhir/model/r4/lib/FHIROperationOutcome.js

```diff
@@ -0,0 +1,117 @@
+/*!
+ * ${copyright}
+ */
+
+// Provides class sap.fhir.model.r4.lib.FHIROperationOutcome
+sap.ui.define([], function () {
+	"use strict";
+
+	/**
+	 * Constructor for a new FHIROperationOutcome. [FHIR Specification]{@link https://www.hl7.org/fhir/operationoutcome.html}
+	 *
+	 * @param {object} oResource The FHIR resource
+	 * @constructs {sap.fhir.model.r4.lib.FHIROperationOutcome} Provides the implementation of a FHIR OperationOutcome
+	 * @alias sap.fhir.model.r4.lib.FHIROperationOutcome
+	 * @author SAP SE
+	 * @protected
```

Comment:
@public because the object should be used

Reply:
done